src/imagery/i.eodag/i.eodag.py

```diff
@@ -879,11 +879,7 @@ def skip_existing(output, search_result):
             if scene_file.exists():
                 creation_time = datetime.utcfromtimestamp(os.path.getctime(scene_file))
                 ingestion_time = scene.properties.get("modificationDate")
-                if (
-                    ingestion_time
-                    and datetime.fromisoformat(ingestion_time).replace(tzinfo=None)
-                    <= creation_time
-                ):
+                if normalize_time(ingestion_time) <= creation_time:
                     # This is to check that the file was completely downloaded
                     # without interruptions.
                     # The reason this works:
```

Comment:
There is also a change that needs to be done to line 892, in accordance with this EODAG change:  https://github.com/CS-SI/eodag/pull/1023/files

Could you please change lines [891, 893] to be:
```python
hashed_file = (
    downloaded_dir / md5((scene.product_type+"-"+scene.properties['id']).encode()).hexdigest()
)
```
To align with the new hashing method.